integration_tests/README.md

```diff
@@ -1,23 +1,90 @@
-###  Run the integration tests
+### Overview
+1. Prerequisites
+1. Configure credentials
+1. Setup Postgres (optional)
+1. Setup virtual environment
+1. Installation for development
+1. Run the integration tests
+1. Run tests
+1. Creating a new integration test
+
+### Prerequisites
+- python3
+- Docker
+
+### Configure credentials
+Edit the env file for your TARGET in `integration_tests/.env/[TARGET].env`.
```

Comment:
Do we want to consolidate all the `integration_tests/.env/[TARGET].env` files into a single `test.env` file instead?